app/views/browse/_relation_member_frame.html.erb

```diff
@@ -0,0 +1,13 @@
+<turbo-frame id="<%= frame_id %>">
+  <% if relation.redacted? && !params[:show_redactions] %>
```

Comment:
Can this ever happen? If it's redacted and we're not showing redactions then we won't have generated the frame will we?

I guess there's an issue that somebody could request the frame manually to try and see the redactions but the controller could just reject requests for a frame for a redacted relation unless the show redactions flag is set?

Reply:
The error message was intended for someone trying to directly access the new endpoint. It cannot happen in our UI, because, as you said, the relation version won't show any details except for this message: _Version x of this relation cannot be shown as it has been redacted. Please see Redaction y for details._

So in the OldRelationMembersController, I would have to add something like:

```
deny_access(nil) if relation.redacted? && !params[:show_redactions]
```